lib/web/command.go

```diff
@@ -275,9 +286,106 @@ func (h *Handler) executeCommand(
 
 	runCommands(hosts, runCmd, h.log)
 
+	// Optionally try to compute the command summary.
+	if output := buffer.Export(); output != nil {
+		err := h.computeAndSendSummary(ctx, hosts, output, clt, identity, req, ws)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+	}
+
 	return nil, nil
 }
 
+func (h *Handler) computeAndSendSummary(
+	ctx context.Context,
+	hosts []hostInfo,
+	output map[string][]byte,
+	clt auth.ClientI,
+	identity srv.IdentityContext,
+	req *CommandRequest,
+	ws *syncRWWSConn,
```

Comment:
`syncRWWSConn` is a concrete implementation. You should use `WSConn` interface here.